docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -149,28 +142,27 @@ export KUBECONFIG=${PWD}/${CLUSTER_NAME}/generated/${CLUSTER_NAME}.kind.kubeconf
 kubectl logs -f -n capv-system -l control-plane="controller-manager" -c manager
 ```
 
-It also might be useful to start a shell session on the docker container running the bootstrap cluster by running `docker ps` and then `docker exec -it <container-id> bash` the kind container.
+It is also useful to start a shell session on the docker container running the bootstrap cluster by running `docker ps` and then `docker exec -it <container-id> bash` the kind container.
 
-### Bootstrap cluster fails to come up
+### Bootstrap cluster fails to come up: node(s) already exist for a cluster with the name
 
 ```
 Error: creating bootstrap cluster: executing create cluster: ERROR: failed to create cluster: node(s) already exist for a cluster with the name \"cluster-name\"
 , try rerunning with —force-cleanup to force delete previously created bootstrap cluster
 ```
 
-Cluster creation fails because a cluster of the same name already exists.
-Try running the `eksctl anywhere create cluster` again, adding the `--force-cleanup` option.
+Cluster creation fails because a cluster of the same name already exists. If you are sure the cluster is not being used, try running the `eksctl anywhere create cluster` again, adding the `--force-cleanup` option.
 
 If that doesn't work, you can manually delete the old cluster:
 
 ```bash
 kind delete cluster --name cluster-name
 ```
 
-### Cluster upgrade fails with management cluster on bootstrap cluster
+### Cluster upgrade fails with management components on bootstrap cluster
 
 If a cluster upgrade of a management (or self managed) cluster fails or is halted in the middle, you may be left in a
-state where the management resources (CAPI) are still on the KinD bootstrap cluster on the Admin machine. Right now, you will have to
+state where the management resources (CAPI) are still on the KinD bootstrap cluster on the admin machine. Right now, you will have to
```

Comment:
We're inconsistent with "Admin machine" vs. "admin machine". I prefer "Admin machine" because it seems like a more real thing to me.  Most of the getting started docs use "Admin machine". I'm open to changing it, however.